deployer/service/src/main/java/org/apache/karaf/cave/deployer/service/rest/DeployerRestService.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.karaf.cave.deployer.service.rest;
+
+import org.apache.karaf.cave.deployer.api.DeployerService;
+import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.http.HttpService;
+
+@Component(name = "org.apache.karaf.cave.deployer.rest")
+public class DeployerRestService {
+
+    @Reference
+    private DeployerService deployerService;
+
+    @Reference
+    private HttpService httpService;
+
+    @Activate
+    public void activate(ComponentContext componentContext) throws Exception {
+        DeployerRestApi restApi = new DeployerRestApi(deployerService);
+        DeployerRestServlet featuresGatewayRestServlet = new DeployerRestServlet(restApi);
+        httpService.registerServlet("/cave/deployer/api", featuresGatewayRestServlet, null, null);
```

Comment:
We should name the variable `deployerRestServlet`.

Reply:
+1, copy/paste mistake. I'm fixing.